### DISCUSSION TITLE
编译cls文件出现错误

### DISCUSSION THREAD

**Discussion by yizhinh** [note286/xduts#154] [Q&A] [answered]
![屏幕截图 2024-01-11 144654](https://github.com/note286/xduts/assets/155969559/9bd94b7f-1ceb-43b8-8926-884a3c29ccae)
如图，编译不通过

**Comment by note286** [comment, marked as answer]
https://github.com/note286/xduts/discussions/15

**Reply by yizhinh** [reply]
有没有一些样例可以提供，不知道自己创建.tex文件后该如何编写

**Reply by note286** [reply]
https://github.com/note286/xduts/discussions/40

**Reply by note286** [reply]
我不确定你是否阅读了手册，也就是`xduts.pdf`。
